src/main/kotlin/gg/skytils/skytilsmod/features/impl/dungeons/PartyFinderStats.kt

```diff
@@ -165,64 +175,110 @@ object PartyFinderStats {
                                 remove(null)
                             }
                             component.append(
-                                UTextComponent("§dImportant Items: §7(Hover)\n\n").setHoverText(
-                                    items.joinToString(
-                                        "§8, "
-                                    ).ifBlank { "§c§lNone" }
-                                )
+                                UTextComponent("§dImportant Items: §7(Hover)\n").setHoverText(items.joinToString(
+                                    "§8, "
+                                ).ifBlank { "§c§lNone" })
                             )
                         }
+                    val bloodMobsKilled = (profileData.player_stats.kills["watcher_summon_undead"]?.toInt()
+                        ?: 0) + (profileData.player_stats.kills["master_watcher_summon_undead"]?.toInt() ?: 0)
+
+                    val magicalPower = profileData.inventory.bag.talisman.toMCItems().filterNotNull().map {
+                        val itemId = ItemUtil.getSkyBlockItemID(it)?.let { id ->
+                            when {
+                                id.startsWith("PARTY_HAT_") -> "PARTY_HAT"
```

Comment:
Would probably be smart to have a map with ids and what they should be replaced with.